content/blog/ultimate-windows-development-environment-guide/index.md

```diff
@@ -0,0 +1,612 @@
+
+To use symbolic links from the CLI, you have to first enable developer mode on your install of Windows. To do this, go to your settings app, open "Update & Security," then select in the sidebar "For developers."
+
+![The settings page for "developer mode"](./developer_mode.png)
+
+Once done, you're able to run `mklink`, which provides you the ability to make a symbolic link.
+
+### Usage {#using-mklink}
+
+By default, it creates a soft link from the first argument to the second. 
+```
+mklink Symlink SourceFile
+```
+
+You're also able to add `/D` to make a soft link to a directory:
+```
+mklink /D SymlinkDir SourceFolder
+```
+
+Finally, to make hard links, you use `/H` for files:
+
+```
+mklink /H Symlink SourceFile
+```
+
+And `/J` for folders:
+
+```
+mklink /J SymlinkDir SourceFolder
+```
+
+### GUI Alternative {#link-shell-extension}
+
+While the CLI enables you to make hard and soft symbolic links, it's far from graceful. It'd be ideal to have that functionality baked right into the explorer menu options if used frequently. Luckily for us, there's an app for that! [Link Shell Extension](https://schinagl.priv.at/nt/hardlinkshellext/linkshellextension.html) adds the options to the context menu itself. It's even able to be installed using [Chocolatey](#package-management):
+
+```
+choco install linkshellextension
+```
+
+Simply right click (or [use the <kbd>Shift</kbd> + <kbd>F10</kbd> shortcut](#built-in-keyboard-shortcuts)) and select "Pick Link Source":
+![The context menu with the option highlighted](./pick_source.png)
+
+Then you're able to navigate to the folder you're looking for, right click, and select "Drop as...":
+
+![The context menu with the option highlighted](./drop_as.png)
+
+There are a myriad of options to choose from and should handle any type of symlink you'd need.
+
+# Conclusion
+
+You'll notice that despite the raw power and capabilities that WSL2 will be bringing to us right around the corner, that I didn't touch on it until later in the article. That's because, while it's an amazing toolset to be able to utilize for those that need it, it's not the only thing that you can do to enable your Windows instance to be powerful for development. Windows (and Microsoft as a whole) has come a long way in the past 10 years, and with their continued effort on projects like WSL, VS Code, and the Windows Terminal, the future looks brighter than ever. 
+
+I want to take a moment to stop and appreciate all of the hard work that the folks at Microsoft and everyone involved in the projects mentioned have done to enable the kind of work I do daily. Thank you.
+
+With that, we're closing off the article. If you have any questions or comments, feel free to ring off in the comment box below. Otherwise, we have [our community Discord](https://discord.gg/FMcvc6T) where we talk not only talk Windows, but Linux, macOS, programming, and everything in between. We look forward to seeing you there!
```

Comment:
```suggestion
If you have any questions or comments, feel free to ring off in the comment box below. Otherwise, we have [our community Discord](https://discord.gg/FMcvc6T) where we talk not only talk Windows, but Linux, macOS, programming, and everything in between. We look forward to seeing you there!
```